auth/service-to-service/auth.py

```diff
@@ -16,33 +16,42 @@
 Demonstrates how to send authenticated service-to-service requests, eg
 for Cloud Run or Cloud Functions"""
 
-# [START google_auth_idtoken_serverless]
 # [START functions_bearer_token]
 # [START cloudrun_service_to_service_auth]
-# [START run_service_to_service_auth]
 import urllib
 
 import google.auth.transport.requests
 import google.oauth2.id_token
 
 
-def make_authorized_get_request(service_url):
+def make_authorized_get_request(service_url, audience):
     """
     make_authorized_get_request makes a GET request to the specified HTTP endpoint
-    in service_url (must be a complete URL) by authenticating with the
-    ID token obtained from the google-auth client library.
+    by authenticating with the ID token obtained from the google-auth client library
+    using the specified audience value.
     """
 
+    # [END functions_bearer_token]
+    """
+    # Cloud Run uses your service's hostname as the `audience` value
+    audience = 'https://my-cloud-run-service.run.app/'
+    """
```

Comment:
If folks are expected to uncomment and use `audience`, I think individually commented lines better match the style used in other Python samples.

```suggestion
    # Cloud Run uses your service's hostname as the `audience` value
    # audience = 'https://my-cloud-run-service.run.app/'
```